docs/guide/contributing.md

```diff
@@ -314,7 +314,11 @@ A `pre-push` hook is configured to run Stylelint and ESLint before pushing your
 
 ## Recommended IDE Setup
 
-We recommend using [VSCode](https://code.visualstudio.com/) along with the [Volar extension](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar)
+We recommend using [VSCode](https://code.visualstudio.com/) along with a few [suggested extensions](https://github.com/Kong/kongponents/blob/db8854954eebb0ab62de21263ab6a5ba9bab11eb/.vscode/settings.json) (VSCode should automatically open up a dialog window suggesting to instal recommended extensions when you open the project folder for the first time).
+
+:::warning NOTE
+To utilize all pre-configured VSCode workplace settings for this project, we recommend opening the containing workspace folder directly directly, instead of saving VSCode workplace.
```

Comment:
I don't think we need to talk through VS Code workspace versus folders, etc. We just add the settings file.

Reply:
Some settings don't take effect if the folder is opened through workplace. This particular setting is enabling `vunguyentuan.vscode-css-variables` plugin to look up CSS variables in `design-tokens` package. I think it's useful to leave this comment in place just in case. It does affect developer experience for contributors.

Reply:
Only some **User** settings do not take effect; the settings in the `settings.json` file will take effect, so I believe this paragraph can be safely removed.

Reply:
What if someone is wondering why are they not getting suggestions on CSS variables from design tokens? People are not gonna learn them any time soon. Leaving this suggestion here will not hurt and might save someone 10 minutes of debugging.

Reply:
I'm not sure you understood my comment, but sure. 

Separately, I left another suggestion to reword this and the last sentence of the note is confusing

Reply:
Oh I thought you wanted me to remove the paragraph in the note. Never mind 😅